src/ostree/ot-builtin-config.c

```diff
@@ -88,7 +88,6 @@ ostree_builtin_config (int argc, char **argv, OstreeCommandInvocation *invocatio
 
   if (!strcmp (op, "set"))
     {
-      printf("GROUP NUMBER = %s  %d\n", opt_group, argc);
```

Comment:
Damm! Stupid me. Thanks!

Reply:
Not at all! This happens to everyone. :)